roles/network_plugin/calico/templates/calico.env.j2

```diff
@@ -4,6 +4,11 @@ ETCD_CERT_FILE="{{ calico_cert_dir }}/cert.crt"
 ETCD_KEY_FILE="{{ calico_cert_dir }}/key.pem"
 CALICO_IP="{{ip | default(ansible_default_ipv4.address) }}"
 CALICO_IP6=""
+{% if calico_network_backend is defined and calico_network_backend == 'none' %}
```

Comment:
Which do you not see the use for? calico_network_backend or CLUSTER_TYPE?
CALICO_NETWORKING_BACKEND (which is what calico_network_backend is used for below) sets if Calico will be used for networking and use bird or gobgp or none (which mean something like flannel or a cloud integration is handling networking), if we default it then bird should be the default.
CLUSTER_TYPE is merely a string identifier that we are starting to include that is stored in the datastore, it is available through the `calicoctl version` command and gives a quick way to get an idea of how a cluster is deployed and high level Calico options.